actionview/lib/action_view/test_case.rb

```diff
@@ -39,6 +39,84 @@ def initialize
       end
     end
 
+    module Decoding
+      extend ActiveSupport::Concern
+
+      class_methods do
+        def inherited(descendant)
+          super
+
+          descendant.class_attribute :decoders, default: decoders.dup
+        end
+
+        # Register a callable to decode rendered content for a given MIME type
+        #
+        # Each registered decoder will also define a <tt>#rendered_$MIME</tt> helper
+        # method, where <tt>$MIME</tt> corresponds to the value of the
+        # <tt>mime</tt> argument
+        #
+        # === Arguments
+        #
+        # <tt>mime</tt> - Symbol the MIME Type name for the rendered content
```

Comment:
`mime` here draws direct inspiration from the Action Dispatch counterpart, but might not be completely accurate. While HTML, JSON, and RSS are all mime types, the piece of information that informs the lookup is the `formats:` option that's passed to the `render` call. 

Should this be more accurately re-named to be `format`?

Reply:
I think `format` better describes it.
It also better fits with the `render formats: :rss, ...` call.